molecule/backup/library/check_backup_files_list.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import re
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+argument_spec = {
+    'instance_info': {'required': True, 'type': 'dict'},
+    'files_list': {'required': True, 'type': 'list'},
+    'stateboard': {'required': True, 'type': 'bool'},
+}
+
+
+if __name__ == '__main__':
+    module = AnsibleModule(argument_spec=argument_spec)
+    params = module.params
+
+    files_list = params['files_list']
+    instance_info = params['instance_info']
+    stateboard = params['stateboard']
+
+    # filter out config subdirectories
+    conf_path = os.path.join(instance_info['work_dir'], 'config').strip(os.path.sep)
+    files_list = list(filter(
+        lambda p: p == conf_path or not p.startswith(conf_path),
+        map(lambda path: path.strip(os.path.sep), files_list),
+    ))
```

Comment:
```suggestion
    files_list = list(sorted(filter(
        lambda p: p == conf_path or not p.startswith(conf_path),
        map(lambda path: path.strip(os.path.sep), files_list),
    )))
```

Reply:
This list isn't sorted on `backup_start`, this test checks that the right files are added.

Reply:
But if order of files will be changed, then test will be failed, but it isn't good, because we check only list of files, not order